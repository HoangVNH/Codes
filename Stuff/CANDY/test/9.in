10
5
DDXXV
4
VXXX
8
XDDDDDDX
6
VVVVVX
1
X
2
VV
8
DDDDDVVV
1
D
6
VVVVDD
9
VXXXXVVVX
