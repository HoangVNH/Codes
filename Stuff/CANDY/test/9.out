4
3
6
5
0
0
7
0
4
8
